.travis.yml

```diff
@@ -27,13 +27,11 @@ jobs:
       dist: xenial
       env:
         - TEST_SYMPY="true" SPLIT="2/2"
-    - python: 3.7
+    - python: 3.9
       dist: xenial
-      # At the time of writing this is Python 3.7 but it will become 3.8 when
-      # all of the dependencies are supported on 3.8.
       env:
         - TEST_ASCII="true"
-        - TEST_OPT_DEPENDENCY="matchpy numpy scipy gmpy2 matplotlib aesara llvmlite autowrap cython wurlitzer python-symengine tensorflow numexpr ipython antlr-python-runtime>=4.7,<4.8 antlr>=4.7,<4.8 cloudpickle pyglet pycosat lfortran python-clang lxml"
+        - TEST_OPT_DEPENDENCY="matchpy numpy scipy gmpy2 matplotlib aesara autowrap cython wurlitzer python-symengine tensorflow numexpr ipython antlr-python-runtime>=4.7,<4.8 antlr>=4.7,<4.8 cloudpickle pycosat lxml" # llvmlite lfortran python-clang pyglet
```

Comment:
These aren't installable with 3.9 yet? What about 3.8?

Reply:
Not sure what the problem is. There is a 800 MB download including openjdk sp I tried to remove some packages to see if the size was reduced. (It was, but 9nly to 700 MB)

If I understand it correctly it cannot install that much and dies. However,  right now it is not the optional dependencies that fails...

Reply:
https://github.com/sympy/sympy/issues/21764#issuecomment-1006078052

Reply:
```suggestion
        - TEST_OPT_DEPENDENCY="matchpy numpy scipy gmpy2 matplotlib aesara llvmlite autowrap cython wurlitzer python-symengine tensorflow numexpr ipython antlr-python-runtime>=4.7,<4.8 cloudpickle pyglet pycosat lfortran python-clang lxml"
```

Reply:
In conda-forge `matplotlib = maptlotlib-base + pyqt` which means we can use `matplotlib-base` if we don't need the pyqt backend.